session-signatures/getPkpSessionSigs/src/index.ts

```diff
@@ -0,0 +1,86 @@
+import { getEnv } from "./utils";
+import { LitNodeClient } from "@lit-protocol/lit-node-client";
+import { LitNetwork, ProviderType, LIT_RPC } from "@lit-protocol/constants";
+import {
+  EthWalletProvider,
+  LitAuthClient,
+} from "@lit-protocol/lit-auth-client";
+import { LitAbility, LitPKPResource } from "@lit-protocol/auth-helpers";
+import * as ethers from "ethers";
+import { LocalStorage } from "node-localstorage";
```

Comment:
Nitpick, but convention is to do external imports first, then local imports

```suggestion
import { LitNodeClient } from "@lit-protocol/lit-node-client";
import { LitNetwork, ProviderType, LIT_RPC } from "@lit-protocol/constants";
import {
  EthWalletProvider,
  LitAuthClient,
} from "@lit-protocol/lit-auth-client";
import { LitAbility, LitPKPResource } from "@lit-protocol/auth-helpers";
import * as ethers from "ethers";
import { LocalStorage } from "node-localstorage";

import { getEnv } from "./utils";
```